### PR TITLE
chore(gradle): Fix gradle test memory allocation

### PR DESCRIPTION
**Pull request by jasonmcintosh**

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -2,7 +2,7 @@ fiatVersion=1.45.0
 korkVersion=7.224.0
 kotlinVersion=1.6.21
 org.gradle.parallel=true
-org.gradle.jvmargs=-Xmx6g
+org.gradle.jvmargs=-Xmx4g
```
Comment (dbyron-sf) [inline gradle.properties:5]: github actions still uses 6g, e.g. [here](https://github.com/spinnaker/orca/blob/f6d23180112ad5b9c46f84d8cf6409781f261c11/.github/workflows/pr.yml#L6).

Comment (jasonmcintosh) [reply gradle.properties:5]: THINK i've caught em all now...

Comment (dbyron-sf) [reply gradle.properties:5]: One more in Dockerfile.compile...

**Comment by dbyron-sf** [comment]
@mergifyio update

**Review by dbyron-sf — APPROVED** [review]

**Pull request merged**